examples/rails-webpacker/README.md

```diff
@@ -0,0 +1,79 @@
+# Rails with Webpacker
+
+Setting up Tailwind with [Rails](https://rubyonrails.org/) is really simple.
+
+First you need [Webpacker](https://github.com/rails/webpacker) to use [PostCSS](https://github.com/postcss/postcss)
+
+Then install `tailwindcss` and `@fullhuman/postcss-purgecss` :
+
+```sh
+yarn add tailwindcss @fullhuman/postcss-purgecss
+```
+
+In your `postcss.config.js`, add `tailwindcss` and `@fullhuman/postcss-purgecss` as well:
+
+```js
+// only needed if you want to purge
+const purgecss = require("@fullhuman/postcss-purgecss")({
```

Comment:
I was able to get it working with just the basic instructions at https://tailwindcss.com/docs/optimizing-for-production#basic-usage, without needing to also add `postcss-purgecss`.

My tailwind.config.js:

```js
module.exports = {
 // ...
  purge: {
    enabled: process.env.NODE_ENV === "production"
    content: ['./app/views/**/*.html.erb'],
  },
 // ...
}
```